tests/stubtest_allowlists/py39.txt

```diff
@@ -106,3 +106,83 @@ collections.Generator.gi_running
 collections.Generator.gi_yieldfrom
 collections.Mapping.get  # Adding None to the Union messed up mypy
 collections.Sequence.index  # Supporting None in end is not mandatory
+
+# Exists at runtime, but missing from stubs
+_ast.Tuple.dims
+_collections_abc.AsyncIterable.__class_getitem__
+_collections_abc.Awaitable.__class_getitem__
+_collections_abc.Container.__class_getitem__
+_collections_abc.Iterable.__class_getitem__
+_collections_abc.MappingView.__class_getitem__
+_imp.source_hash
+_thread._ExceptHookArgs.n_fields
+_thread._ExceptHookArgs.n_sequence_fields
+_thread._ExceptHookArgs.n_unnamed_fields
+ast.Tuple.dims
+ast.main
+asyncio.BufferedProtocol.eof_received
+asyncio._enter_task
+asyncio._leave_task
+asyncio._register_task
+asyncio._unregister_task
+asyncio.protocols.BufferedProtocol.eof_received
+collections.AsyncIterable.__class_getitem__
+collections.Awaitable.__class_getitem__
+collections.Container.__class_getitem__
+collections.Iterable.__class_getitem__
+collections.MappingView.__class_getitem__
+distutils.command.bdist_wininst
+distutils.command.bdist_wininst.bdist_wininst
+distutils.dist.DistributionMetadata.set_classifiers
+distutils.dist.DistributionMetadata.set_keywords
+distutils.dist.DistributionMetadata.set_platforms
+distutils.sysconfig.expand_makefile_vars
+distutils.sysconfig.get_python_version
+distutils.util.get_host_platform
+email.headerregistry.MessageIDHeader.max_count
+ftplib.FTP.trust_server_pasv_ipv4_address
+hmac.HMAC.digest_cons
+hmac.HMAC.inner
+hmac.HMAC.outer
+importlib.abc.TraversableResources
+importlib.machinery.PathFinder.find_distributions
+importlib.metadata.EntryPoint.attr
+importlib.metadata.EntryPoint.module
+importlib.metadata.EntryPoint.pattern
+importlib.util.source_hash
+lib2to3.pgen2.tokenize.COLONEQUAL
+multiprocessing.managers.SharedMemoryServer.create
+multiprocessing.managers.SharedMemoryServer.list_segments
+multiprocessing.managers.SharedMemoryServer.public
+multiprocessing.managers.SharedMemoryServer.release_segment
+multiprocessing.managers.SharedMemoryServer.shutdown
+multiprocessing.managers.SharedMemoryServer.track_segment
+pstats.FunctionProfile
+pstats.Stats.get_stats_profile
+pstats.StatsProfile
+pyexpat.XMLParserType.SkippedEntityHandler
+pyexpat.XMLParserType.intern
+re.Pattern.scanner
+types.CoroutineType.cr_origin
+types.GenericAlias.__call__
```

Comment:
Same as in the py310 entry, I'd put this in a "Won't fix" section, with a ref to #6392